### PR TITLE
Fix the direction from only going backwards

### PR DESCRIPTION
**Pull request by zenangst**

### CODE REVIEW

File: Source/PagesController.swift [hunk 1]
```diff
@@ -40,10 +40,11 @@ extension PagesController {
 
   public func goto(index: Int) {
```
Comment (3lvis) [inline Source/PagesController.swift]: Why is this `goto` instead of `goTo`?

File: Source/PagesController.swift [hunk 2]
```diff
@@ -40,10 +40,11 @@ extension PagesController {
...
     if index > -1 && index < pages.count {
+      let direction: UIPageViewControllerNavigationDirection = (index > currentIndex) ? .Forward : .Reverse
```
Comment (vadymmarkov) [inline Source/PagesController.swift:43]: Do we need to add type annotation `UIPageViewControllerNavigationDirection` to be able to use `.Forward` and `.Reverse`, right?

File: Source/PagesController.swift (hunk 1 above)
Comment (zenangst) [inline Source/PagesController.swift]: @3lvis Used http://en.wikipedia.org/wiki/Goto for reference

File: Source/PagesController.swift (hunk 2 above)
Comment (zenangst) [inline Source/PagesController.swift:43]: @vadymmarkov You need to declare type when using the shorthands such as `.Forward`

File: Source/PagesController.swift [hunk 3]
```diff
@@ -40,10 +40,11 @@ extension PagesController {
 
   public func goto(index: Int) {
     if index > -1 && index < pages.count {
```
Comment (vadymmarkov) [inline Source/PagesController.swift]: Maybe `index >= 0` is better? Just don't like to use -1 :smiley:

File: Source/PagesController.swift (hunk 1 above)
Comment (zenangst) [inline Source/PagesController.swift]: @3lvis fixed

File: Source/PagesController.swift (hunk 3 above)
Comment (zenangst) [inline Source/PagesController.swift]: @vadymmarkov fixed :)

**Comment by vadymmarkov** [comment]
Merge?

**Comment by 3lvis** [comment]
Merge!

**Comment by zenangst** [comment]
:scream: TOO MUCH DRAMA!

**Pull request merged**